code/modules/spells/spell_types/lesserlichdom.dm

```diff
@@ -0,0 +1,151 @@
+	for(var/mob/M in targets)
+		var/list/hand_items = list()
+		if(iscarbon(M))
+			hand_items = list(M.get_active_held_item(),M.get_inactive_held_item())
+		if(!hand_items.len)
+			to_chat(M, "<span class='caution'>You must hold an item you wish to make your phylactery...</span>")
+			return
+		if(!M.mind.hasSoul)
+			to_chat(user, "<span class='caution'>You do not possess a soul.</span>")
+			return
+
+		var/obj/item/marked_item
+
+		for(var/obj/item/item in hand_items)
+			// I ensouled the nuke disk once. But it's probably a really
+			// mean tactic, so probably should discourage it.
+			if((item.item_flags & ABSTRACT) || HAS_TRAIT(item, TRAIT_NODROP) || SEND_SIGNAL(item, COMSIG_ITEM_IMBUE_SOUL, user))
+				continue
+			marked_item = item
+			to_chat(M, "<span class='warning'>You begin to focus your very being into [item]...</span>")
+			break
+
+		if(!marked_item)
+			to_chat(M, "<span class='warning'>None of the items you hold are suitable for emplacement of your fragile soul.</span>")
+			return
+
+		playsound(user, 'sound/effects/pope_entry.ogg', 100)
+
+		if(!do_after(M, 50, needhand=FALSE, target=marked_item))
+			to_chat(M, "<span class='warning'>Your soul snaps back to your body as you stop ensouling [marked_item]!</span>")
+			return
+
+		marked_item.name = "lesser ensouled [marked_item.name]"
+		marked_item.desc += "\nA terrible aura surrounds this item, its very existence is offensive to life itself..."
+		marked_item.add_atom_colour("#187918", ADMIN_COLOUR_PRIORITY)
+
+		new /obj/item/lesserphylactery(marked_item, M.mind)
+
+		to_chat(M, "<span class='userdanger'>With a hideous feeling of emptiness you watch in horrified fascination as skin sloughs off bone! Blood boils, nerves disintegrate, eyes boil in their sockets! As your organs crumble to dust in your fleshless chest you come to terms with your choice. You're a lesser lich!</span>")
+		M.mind.hasSoul = FALSE
+		M.mind.hellbound = TRUE // No revival other than lichdom revival
+		M.set_species(/datum/species/skeleton)
+		// no robes spawn for a lesser spell
+		// you only get one phylactery.
+		M.mind.RemoveSpell(src)
+
+
+/obj/item/lesserphylactery
+	name = "lesser phylactery"
+	desc = "Stores souls. Revives lesser liches. Also repels mosquitos. Can only be used to revive a lich twice."
+	icon = 'icons/obj/projectiles.dmi'
+	icon_state = "bluespace"
+	color = "#187918"
+	light_color = "#187918"
+	light_system = MOVABLE_LIGHT
+	light_range = 3
+	var/lon_range = 3
+	var/resurrections = 0
+	var/datum/mind/mind
+	var/respawn_time = 3600  //Double the time of a regular phylactery
+
+	var/static/active_phylacteries = 0
+
+/obj/item/lesserphylactery/Initialize(mapload, datum/mind/newmind)
+	. = ..()
+	mind = newmind
+	name = "lesser phylactery of [mind.name]"
+
+	active_phylacteries++
+	GLOB.poi_list |= src
+	START_PROCESSING(SSobj, src)
+	if(initial(SSticker.mode.round_ends_with_antag_death))
+		SSticker.mode.round_ends_with_antag_death = FALSE
```

Comment:
what's the point of this?

Reply:
Spell is also in the wizard spellbook, I'll add a check for if its an antag so regular chaplains dont end up toggling this

Reply:
I am confused what this toggle does and why you are toggling it

Reply:
Its the round end toggle, if the round is meant to toggle on antag death, it stops the round ending, as the lich would technically be dead but awaiting phylactery revival.
I've put in a special_role check so this only toggles if an antag creates a phylatery.
tl;dr - no round end on antag death, toggles back to end on antag death if phylactery gets destroyed

Reply:
I'll remove lesser lichdom from wizard spellbook, and remove the toggle